client/src/pages/Search/SearchHeader/ViewToggler.js

```diff
@@ -0,0 +1,35 @@
+import { Button } from "@mui/material";
+import React from "react";
+import LocationOnIcon from "@mui/icons-material/LocationOn";
+import ViewModuleIcon from "@mui/icons-material/ViewModule";
+
+export default function ViewToggler({ viewMode, setViewMode }) {
+  const NotSelectedStyle = {
+    color: "grey",
+  };
+
+  return (
+    <>
```

Comment:
let's center this and have some spacing between them like this:
![image](https://user-images.githubusercontent.com/51217000/227807957-695669c7-3d43-4f79-8c4c-d5e47b8acd22.png)